core/src/components/datetime/datetime.md.scss

```diff
@@ -140,3 +140,23 @@
 :host .datetime-view-buttons ion-button {
   color: $text-color-step-200;
 }
+
+.calendar-month-year-toggle {
+  @include padding(12px, 16px, 12px, calc(var(--padding-start) + var(--ion-safe-area-left, 0px)));
+
+  min-height: 48px;
+}
+
+.calendar-month-year-toggle ion-ripple-effect {
+  color: currentColor;
+}
+
```

Comment:
The focus state seems to be missing now:

| `main` | `FW-3547` |
| ---| ---|
| ![main](https://github.com/ionic-team/ionic-framework/assets/6577830/ecf3f899-bc95-4a51-9f74-6e87e1d7ecde) | ![FW-3547](https://github.com/ionic-team/ionic-framework/assets/6577830/bdb06bc8-2c3a-4d4e-a469-1856c65e1137) |

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/28443/commits/e8e93cb8a269c6870dc62bbb0420233825bc0ea7